site_config

```diff
@@ -0,0 +1,23 @@
+server {
+        listen 80;
+        listen [::]:80;
```

Comment:
Why are we mentioning same port two times?

Reply:
> Why are we mentioning same port two times?

The second one is for ipv6 configuration, if required, it will be used